boards/BMS/Inc/App/App_EepromExitCode.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+/**
+ * @brief  EEPROM Status structures definition
+ */
+typedef enum
+{
+    EEPROM_OK           = 0x00U,
+    EEPROM_ADDR_ERROR   = 0x01U,
+    EEPROM_I2C_ERROR    = 0x02U,
+    EEPROM_SIZE_ERROR   = 0x03U,
+    EEPROM_DATA_CORRUPT = 0x04U
+} EEPROM_StatusTypeDef;
```

Comment:
Not a huge fan of having this in it's own file, but since I wanted it to be used in IO, APP, and Test files, this seemed the easiest way. Put in App header directory similar to App_SharedExitCode.h